release/scripts/perform.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env sh
+
+#set -x
```

Comment:
Removed. `set -e` is the default on all current shells I think.

Reply:
At least on my shell it doesn't seem to be the default, just ran a quick test: 

```
➜  tmp ./test.sh
Running with set -e
error: no such subcommand: `release`
➜  tmp vi test.sh
➜  tmp ./test.sh
Running without set -e
error: no such subcommand: `release`
This is after a failed command.
➜  tmp
```

I think we should err on the safe side and explicitly set this, as otherwise a branch and potentially a PR would be created with a wrong version number since cargo release didn't increment it (for that failure scenario).